Updated my git-practice-v2

source-duck opened the pull request:
Updated index.html
Added advice/source-duck-advice.html, css/source-duck.css, added img directory and several images.
Resolved merge conflict Jill-Anderson-advice.html, by adding both.

twosavoie commented:
Hi @source-duck,

Congrats on making your first pull request! 

This repo is shared by all Skillcrush students and adding images would mean that it would become too large quickly. So, unfortunately, in this exercise we wouldn’t be able to merge your code into to the Skillcrush repo. You can link to an image outside of the Skillcrush repo by adding it’s url to your CSS. You can refer how to add an image in [this lesson](https://learn.skillcrush.com/module-4/how-images-attributes-work/).

Once you've removed the img folder from the advice branch, you can add the changes by running `git add -A` (the capital A is important), commit as you usually would, and push the advice branch to GitHub. Your pull request will be updated automatically. 

Let us know if you have any questions!

Best,
Lisa

source-duck commented:
Hi @twosavoie,
Thank you for the correction guidance!

Removed img directory, updated images via absolute url's. YAY!
Added, committed, resolved merge conflict, merged.

Again Thank You!
Jogee

twosavoie commented:
Hi @source-duck,

Great! The changes have been made to the main branch. The next step would be to merge that change into the advice branch. You can follow the directions in the lesson where your local main branch is merged into your local advice branch.

Once the merge is complete, push your advice branch to GitHub your PR will be updated and we'll take a look!

Best,
Lisa

twosavoie commented:
Hi @source-duck,

Great job removing the images - that's awesome!

It looks like there is some commented-out CSS in the index.html file. Since this file is used by all students, can you remove it please? Then add, commit, and push your code to GitHub and we'll take a look!

Best,
Lisa

source-duck commented:
Hi @twosavoie,

Thank you for continuing to share your time and guidance! Updated index.html, removing the css code block that I forgot to remove before making the PR (apologies for causing extra work/steps).
Cheers,
Jogee

twosavoie commented:
Hi @source-duck,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

Best,
Lisa
